### ISSUE TITLE
Readme.md copied from neural-doodle 

### ISSUE THREAD

**Issue by AxelStrem** [alexjc/neural-enhance#105]
At the "Manual Installation" section, it says
`You'll also need to download this pre-trained neural network (VGG19, 80Mb) and put it in the same folder as the script to run.`
with a link to neural-doodle's VGG19.
  Took me a while to realize that's not what I actually need:D

**Comment by alexjc** [comment]
You need `VGG19` for training a super-resolution yourself. Maybe add this to the `README`:
```If you want to train your own network, you'll also need [...]```

**Comment by AxelStrem** [comment]
@alexjc Oh ok, I didn't realize it was there for training. It just says "Manual Installation" and doesn't mention the pre-trained networks from releases, but perhaps that's okay, it might be just me being slow:)

**Comment by alexjc** [comment]
You're right it's not clear, I'll fix it when I next update the code if anyone hasn't made a PR by then.
